### ISSUE TITLE
debounce() is now gone

### ISSUE THREAD

**Issue by Daxito** [SwifterSwift/SwifterSwift#868]
Hi, I can't find 
debounce()
function anymore and nobody says anything about its removal, what is the new function named?

Thanks

**Comment by guykogus** [comment]
Looks like it was removed in 42353f5787fc946e0936aed014b4d88f3a4d327d, when we moved to version 5.0.0. It seems like we got rid of the global `SwifterSwift` struct. What do you think about moving this to be an extension of `DispatchQueue`? E.g.
```Swift
func debounce(millisecondsDelay: Int, action: @escaping () -> Void) -> () -> Void {
    // http://stackoverflow.com/questions/27116684/how-can-i-debounce-a-method-call
    var lastFireTime = DispatchTime.now()
    let dispatchDelay = DispatchTimeInterval.milliseconds(millisecondsDelay)
    return {
        self.asyncAfter(deadline: lastFireTime + dispatchDelay) {
            let now = DispatchTime.now()
            if now >= lastFireTime + dispatchDelay {
                lastFireTime = now
                action()
            }
        }
    }
}
```

**Comment by Daxito** [comment]
That works for me! thank you! and great extensions!

**Comment by guykogus** [comment]
@Daxito fixed by #869
